bsdlib/include/nrf_socket.h

```diff
@@ -1264,7 +1300,7 @@ int nrf_select(int                        nfds,
  * @brief Method to poll for events on one or more sockets.
  *
  * @param[inout] An array of sockets, and respective for each socket that the caller polls for.
```

Comment:
```suggestion
 * @param[in,out] An array of sockets, and respective for each socket that the caller polls for.
```

Reply:
ok, changing also one inout to in,out later in file (p_optval)